source/draw/AbstractContext.ooc

```diff
@@ -36,4 +36,6 @@ AbstractContext: abstract class {
 	alignWidth: virtual func (width: Int, align := AlignWidth Nearest) -> Int { width }
 	update: abstract func
 	isAligned: virtual func (width: Int) -> Bool { true }
+	_createContext := static func -> This { CpuContext new() }
```

Comment:
I wanted `_createContext` to be a static data member, Closure which you can reassign later. I think in your example above one cannot reassign anything to `_createContext`.